repos/known-good.proj

```diff
@@ -35,6 +35,7 @@
         <RepositoryReference Include="common" />
         <RepositoryReference Include="netcorecli-fsc" />
         <RepositoryReference Include="newtonsoft-json" />
+        <RepositoryReference Include="newtonsoft-json901" />
```

Comment:
Was core-setup also one that depended on 9.0.1 or was that not a thing anymore?

Reply:
It did have a dependency afaik

Reply:
There should be a `RepositoryReference` in repos/core-setup as well then.